docs/source/development/commands_reference.md

```diff
@@ -316,7 +316,7 @@ Call the `run()` method of the `KedroSession` defined in `kedro.framework.sessio
 kedro run
 ```
 
-`KedroContext` can be extended in `run.py` (`src/<package_name>/run.py`). In order to use the extended `KedroContext`, you need to set `context_path` in the [`pyproject.toml` configuration file](../faq/architecture_overview.md#kedro-project).
+`KedroContext` can be extended in `run.py` (`src/<package_name>/run.py`). In order to use the extended `KedroContext`, you need to set `context_path` in the `pyproject.toml` configuration file.
```

Comment:
Maybe link to https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html ?

Reply:
I don't know @astrojuanlu. I'd be happy to add it, but I looked at the page and `context_path` isn't mentioned there because it covers `settings.py`. Perhaps that page itself needs to be extended to cover `pyproject.toml` as well? cc @merelcht

Reply:
Makes sense, and I've created an issue for that very thing.

Reply:
xref gh-2377 👍🏽